Hestia/app/src/main/java/hestia/UI/ExpandableListAdapter.java

```diff
@@ -105,7 +105,12 @@ public boolean onMenuItemClick(MenuItem item) {
                                 //Settings
                                 break;
                             case R.id.delete:
-                                //c.deleteDevice((Device)((DeviceBar) getChild(groupPosition, childPosition)).getDevice())
+                                /**
```

Comment:
I would suggest removing this comment before merging into development. That way, we will keep our development branch clean.